servers/quarkus-server/pom.xml

```diff
@@ -42,6 +42,19 @@
       <artifactId>nessie-services</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.projectnessie</groupId>
+      <artifactId>nessie-jaxrs</artifactId>
```

Comment:
Do we really need `nessie-jaxrs` with this scope in quarkus-server?
(I don't see any production code in `nessie-jaxrs`)

Reply:
good catch, we don't need this and it was a leftover